tests/CMakeLists.txt

```
@@ -0,0 +1,23 @@
+enable_testing()
+find_package(GTest 1.10 REQUIRED)
```

```suggestion
find_package(GTest REQUIRED)
```

See above about avoiding overspecifying versions (Do we care if we find a compatible googletest > 1.10? Probably not.).